File: Retos/reto2/c++/adivinanza.cpp
#include <iostream>
#include <cstdlib>
#include <ctime>

using namespace std;

int main()
{

    int numero, numero_usuario, intentos = 0;

    cout << "Adivina el numero del 1 al 100\n \n"
         << endl;

    srand(time(NULL));
    numero = rand() % 100 + 1;

    do
    {
        cout << "Ingrese un numero: ";
        cin >> numero_usuario;
        intentos++;

        if (numero_usuario < numero)
        {
            cout << "El numero es mayor\n";
        }
        else if (numero_usuario > numero)
        {
            cout << "El numero es menor\n";
        }
        else
        {
            cout << "Felicidades, adivinaste el numero en " << intentos << " intentos\n";
        }

    } while (numero_usuario != numero);

    return 0;
}

File: Retos/reto2/adivinanza.py
import random

# Genera un numero aleatorio entre 1 y 100
numero = random.randint(1, 100)

print("Adivina un numero entre 1 y 100")

# Recibe un numero del usuario
numero_usuario = int(input("Ingrese un numero: "))


while numero!= numero_usuario:
    if numero < numero_usuario:
        print("El numero es menor")
        numero_usuario = int(input("Ingrese un numero: "))
    elif numero > numero_usuario:
        print("El numero es mayor")
        numero_usuario= int(input("Ingrese un numero: "))
    else:
        break

print("Felicidades, adivinaste el numero")

